src/main/java/org/sagebionetworks/web/client/EntityTypeUtils.java

```diff
@@ -46,6 +47,10 @@ public static String getEntityClassNameForEntityType(String entityType) {
 		return className;
 	}
 
+	public static EntityType getEntityType(EntityHeader header) {
+		return getEntityTypeForEntityClassName(header.getType());
+	}
```

Comment:
Shortcut for `getIcon/EntityTypeForEntityClassName(entityHeader.getType())`

Need a test or two